eZ/Publish/Core/settings/repository.yml

```diff
@@ -24,6 +24,19 @@ parameters:
 
     # Signal Slot API wrapper
     ezpublish.signalslot.repository.class: eZ\Publish\Core\SignalSlot\Repository
+    ezpublish.signalslot.service.content.class: eZ\Publish\Core\SignalSlot\ContentService
+    ezpublish.signalslot.service.content_type.class: eZ\Publish\Core\SignalSlot\ContentTypeService
+    ezpublish.signalslot.service.field_type.class: eZ\Publish\Core\SignalSlot\FieldTypeService
+    ezpublish.signalslot.service.role.class: eZ\Publish\Core\SignalSlot\roleService
```

Comment:
Good catch. But still wonder why it works on my local machine. That's odd. It should scream about this typo from the very beginning. I'll setup new environment and run tests again.